js/scenes/TestScreen/index.js

```diff
@@ -0,0 +1,25 @@
+import { Button, Text } from 'react-native';
+import { colors } from './../../styles/baseStyles';
+import React, { Component } from 'react';
+import { sendEmail } from '../../helpers/email';
+
+const message = {
+  receiver: 'Djordje_V@hotmail.com',
+  sender: 'ringong95@gmail.com',
+  subject: 'in app send?',
+  receiverFName: 'timmy',
+  receiverLName: 'jimmy'
+};
```

Comment:
We probably want a function that (given some data) creates this object. That way poor Djordje doesn't get spammed.